### PR TITLE
feat(api-docs): Add README

### PR DESCRIPTION
**Pull request by ceorourke**
Add a README to explain a bit more about what this repo is used for and how to use it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Sentry's API Schema
+
+Sentry's public API schema uses [OpenAPI](https://swagger.io/specification/) v3.0.1.
+
+This repository is just an artifact and the source of truth can be found in [Sentry](https://github.com/getsentry/sentry/tree/master/api-docs).
+
+# Usage
```
Comment (manuzope) [inline README.md:7]: Not valid anymore cause those files don't exist :)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Sentry's API Schema
+
+Sentry's public API schema uses [OpenAPI](https://swagger.io/specification/) v3.0.1.
+
+This repository is just an artifact and the source of truth can be found in [Sentry](https://github.com/getsentry/sentry/tree/master/api-docs).
```
Comment (manuzope) [inline README.md:5]:
```suggestion
This repository is an artifact used while deploying the API schema. The source of truth can be found in [Sentry](https://github.com/getsentry/sentry/tree/master/api-docs), along with the [tests](example.com).
```

Comment (manuzope) [reply README.md:5]: Maybe also throw in "Do not create PRs against this repo"

File: README.md (hunk 1 above)
Comment (ceorourke) [reply README.md:7]: @manuzope 🤔  they don't? I just pulled and I still see them all [here](https://github.com/getsentry/sentry/tree/master/api-docs)

Comment (ceorourke) [reply README.md]: nvm chatted off-GH about this and clarified

**Review by manuzope — APPROVED** [review]

**Pull request merged**